glue/plugins/wcs_autolinking/tests/test_wcs_autolinking.py

```diff
@@ -37,30 +37,31 @@ def test_wcs_autolink_emptywcs():
     assert len(links) == 0
 
 
-def test_wcs_autolink_dimensional_mismatch():
-
-    # No links should be found because the WCS don't actually have well defined
-    # physical types.
-
-    wcs1 = WCS(naxis=1)
-    wcs1.wcs.ctype = ['FREQ']
-    wcs1.wcs.set()
-
-    data1 = Data()
-    data1.coords = wcs1
-    data1['x'] = [1, 2, 3]
-
-    wcs2 = WCS(naxis=3)
-    wcs2.wcs.ctype = 'DEC--TAN', 'FREQ', 'RA---TAN'
-    wcs2.wcs.set()
-
-    data2 = Data()
-    data2.coords = wcs2
-    data2['x'] = np.ones((2, 3, 4))
-
-    dc = DataCollection([data1, data2])
-    links = wcs_autolink(dc)
-    assert len(links) == 0
+# def test_wcs_autolink_dimensional_mismatch():
+    """
+    No links should be found because the WCS don't actually have well defined
+    physical types.
+    """
+
+    # wcs1 = WCS(naxis=1)
+    # wcs1.wcs.ctype = ['FREQ']
+    # wcs1.wcs.set()
+    #
+    # data1 = Data()
+    # data1.coords = wcs1
+    # data1['x'] = [1, 2, 3]
+    #
+    # wcs2 = WCS(naxis=3)
+    # wcs2.wcs.ctype = 'DEC--TAN', 'FREQ', 'RA---TAN'
+    # wcs2.wcs.set()
+    #
+    # data2 = Data()
+    # data2.coords = wcs2
+    # data2['x'] = np.ones((2, 3, 4))
+
+    # dc = DataCollection([data1, data2])
+    # links = wcs_autolink(dc)
+    # assert len(links) == 0
```

Comment:
Can you just remove this code since indeed the test is no longer needed?

Reply:
Sure